src/main/webapp/WEB-INF/jsp/calendarNarrowView.jsp

```diff
@@ -24,8 +24,8 @@
 <jsp:directive.include file="/WEB-INF/jsp/css.jsp"/>
 <c:set var="n"><portlet:namespace/></c:set>
 
-<portlet:actionURL var="showDatePickerURL" escapeXml="false"><portlet:param name='action' value='showDatePicker'/><portlet:param name='show' value='true'/></portlet:actionURL>
-<portlet:actionURL var="hideDatePickerURL" escapeXml="false"><portlet:param name='action' value='showDatePicker'/><portlet:param name='show' value='false'/></portlet:actionURL>
+<portlet:renderURL var="showDatePickerURL" escapeXml="false"><portlet:param name='action' value='showDatePicker'/><portlet:param name='show' value='true'/></portlet:renderURL>
+<portlet:renderURL var="hideDatePickerURL" escapeXml="false"><portlet:param name='action' value='showDatePicker'/><portlet:param name='show' value='false'/></portlet:renderURL>
```

Comment:
Is this change okay?  Does the portlet remember your show/hide choice as a portlet preference?  (I seem to remember that maybe it does???)

Portlet preferences may only be changed during an action phase, iirc.

Reply:
Good catch. This is used with `$.ajax()` (which defaults to `GET`), but it's an action mapping in the controller. I'll flip these back to action URLs and update set `method: POST` on the AJAX call.